workflow/workflow_commands.go

```diff
@@ -1445,6 +1448,76 @@ func TraceWorkflow(c *cli.Context) error {
 	return nil
 }
 
+func UpdateWorkflow(c *cli.Context) error {
+	wid := c.String(common.FlagWorkflowID)
+	rid := c.String(common.FlagRunID)
+	name := c.String(common.FlagName)
+	uid := c.String(common.FlagUpdateID)
```

Comment:
It looks like we tell users that exactly one of FlagName or FlagUpdateID must be provided, but then we also ignore did, and we verify that FlagName must be non-empty. Why don't we just not add FlagUpdateID until it's ready?